home.css

```diff
@@ -0,0 +1,643 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+}
```

Comment:
- [ ] Dear @prius2055, as a frontend engineer we are expected to deliver a webpage that is almost identical to the original design. And for that, we need to follow the exact same colors, for text, backgrounds, borders, etc, from the original design. Please use the appropriate font `Fonts used originally: Cocogoose, Lato`.

Reply:
Resolved, thanks